src/include/daos_srv/daos_server.h

```diff
@@ -338,6 +338,8 @@ enum dss_ult_type {
 	DSS_ULT_DRPC,
 	/** miscellaneous ULT */
 	DSS_ULT_MISC,
+	/* blob I/O module commands */
+	DSS_ULT_BIO,
```

Comment:
I prefer to use existing ULT type for this moment, and make a cleanup patch to re-organize the ULT types later, so let's just use REBUILD or AGGREGATE for this moment.

Reply:
ok, I will add TODO comment in code for this, and switch back to AGGREGATE for now